packages/react-devtools-extensions/build.js

```diff
@@ -56,14 +56,7 @@ const build = async (tempPath, manifestPath) => {
   const binPath = join(tempPath, 'bin');
   const zipPath = join(tempPath, 'zip');
 
-  const webpackPath = join(
-    __dirname,
-    '..',
-    '..',
-    'node_modules',
-    '.bin',
-    'webpack',
-  );
+  const webpackPath = join(__dirname, 'node_modules', '.bin', 'webpack');
```

Comment:
Not sure why we were using `webpack` from root `node_modules`.

Updated to use it from `react-devtools-extensions` package, where `webpack`'s version was updated. The root one is still using version 4.x.x.